Kitodo/src/main/java/org/kitodo/production/services/data/RulesetService.java

```diff
@@ -223,11 +229,24 @@ private QueryBuilder getRulesetsForCurrentUserQuery() {
      *            database object that references the ruleset
      * @return a Ruleset Management in which the ruleset has been loaded
      */
-    public RulesetManagementInterface openRuleset(Ruleset ruleset) throws IOException {
-        String rulesetPath = Paths.get(ConfigCore.getParameter(ParameterCore.DIR_RULESETS), ruleset.getFile())
-                .toString();
-        RulesetManagementInterface openRuleset = ServiceManager.getRulesetManagementService().getRulesetManagement();
-        openRuleset.load(new File(rulesetPath));
-        return openRuleset;
+    public RulesetManagementInterface openRuleset(Ruleset ruleset) throws IOException, RulesetNotFoundException {
```

Comment:
Could the current `openRuleset` method be removed and `openRulesetFile` be renamed to `openRuleset`, since the private `openRulesetFile` method is currently only be called in `openRuleset`?